runtime/src/lib.rs

```diff
@@ -1064,9 +1044,7 @@ construct_runtime!(
         Offences: pallet_offences,
         Historical: pallet_session_historical::{Pallet},
         ImOnline: pallet_im_online,
-        SettlementFFlonkPallet: pallet_fflonk_verifier,
         Poe: pallet_poe,
-        SettlementZksyncPallet: pallet_zksync_verifier,
```

Comment:
I'm scared... I was sure that we put the indexes also in the solo-chain runtime.... But I was wrong. I'm not sure about what can be the clients side effect of this changes.

If we did this to be applied in the next regenesis maybe is better to take also the occasion of add also the indexes here (not in this PR ... indeed)

Reply:
We're going to get rid of everything related to solo chain and keep only relay, I think we can skip